statsmodels/base/model.py

```diff
@@ -735,10 +735,15 @@ def predict(self, exog=None, transform=True, *args, **kwargs):
 
         Returns
         -------
-        prediction : ndarray or pandas.Series
+        prediction : ndarray, pandas.Series or pandas.DataFrame
             See self.model.predict
 
         """
+        from copy import copy
+        import pandas as pd
+
+        orig_exog = copy(exog)
```

Comment:
do we actually need to make a copy, or do we just need some information from exog that we should keep?